client/src/components/GraduateForm.js

```diff
@@ -1,48 +1,67 @@
-import React, { useState } from "react";
-import { Formik, Form } from "formik";
-import FormField from "../constant/FormField";
+import React, { useContext } from 'react';
+import { Formik, Form } from 'formik';
 import * as Yup from "yup";
-import styled from "styled-components";
-import StyledButton from "../constant/StyledButton";
+import FormField from "../constant/FormField";
+import StyledButton from '../constant/StyledButton';
+import { ProfileContext } from '../context/ProfileContext';
+
+import styled from 'styled-components';
 
 
 const GraduateForm = () => {
-	const [state, setState] = useState (null);
+	const { profiles, addProfile, profile, editProfile }  = useContext(ProfileContext);
+	console.log('add', profile);
 
-	const handleSubmit = ({ values,actions }) => {
-		console.log("values", values, "actions", actions);
-		// setTimeout(() => {
-		// 	alert(JSON.stringify(values, null, 2));
-		// 	actions.setSubmitting(false);
-		// }, 1000);
+	const handleSubmit = (values) => {
+		const { name, email, website, phone } = values;
+		const newProfile ={
+			'full_name':name,
```

Comment:
These naming inconsistencies can cause countless bugs and frustrations down the line... is it worth just committing to one name, and refactoring the rest of the code to use that convention elsewhere? What is considered the "true" model?